### ISSUE TITLE
Exception when closing Protégé

### ISSUE THREAD

**Issue by csnyulas** [lukasged/oops-plugin#7]
When I exit Protégé, after having activated the "OOPS! Evaluator Tab", I get the following exception thrown 3 times:

```
*** WARNING BADLY BEHAVING LISTENER: oops.ui.OOPSClassHierarchyViewComponent$$Lambda$124/948652539 *** 
    Possible culprit (trace from when listener was added):  
          oops.ui.OOPSClassHierarchyViewComponent.performExtraInitialisation(OOPSClassHierarchyViewComponent.java:45) 
          org.protege.editor.owl.ui.view.AbstractOWLEntityHierarchyViewComponent.initialiseView(AbstractOWLEntityHierarchyViewComponent.java:76) 
          org.protege.editor.owl.ui.view.AbstractOWLSelectionViewComponent.initialiseOWLView(AbstractOWLSelectionViewComponent.java:103) 
          org.protege.editor.owl.ui.view.AbstractOWLViewComponent.initialise(AbstractOWLViewComponent.java:43) 
          org.protege.editor.core.ui.view.View.createContent(View.java:424) 
          org.protege.editor.core.ui.view.View.createUI(View.java:217) 
          org.protege.editor.core.ui.view.View$1.hierarchyChanged(View.java:123) 
          java.awt.Component.processHierarchyEvent(Component.java:6692) 
          java.awt.Component.processEvent(Component.java:6311) 
          java.awt.Container.processEvent(Container.java:2234) 
          java.awt.Component.dispatchEventImpl(Component.java:4881) 
          java.awt.Container.dispatchEventImpl(Container.java:2292) 
          java.awt.Component.dispatchEvent(Component.java:4703) 
          java.awt.Component.createHierarchyEvents(Component.java:5541) 
          java.awt.Container.createHierarchyEvents(Container.java:1443) 
          java.awt.Container.createHierarchyEvents(Container.java:1439) 
          java.awt.Container.createHierarchyEvents(Container.java:1439) 
          java.awt.Container.createHierarchyEvents(Container.java:1439) 
          java.awt.Container.createHierarchyEvents(Container.java:1439) 
          java.awt.Container.createHierarchyEvents(Container.java:1439) 
          java.awt.Component.show(Component.java:1622) 
          java.awt.Component.show(Component.java:1654) 
          java.awt.Component.setVisible(Component.java:1606) 
          javax.swing.JComponent.setVisible(JComponent.java:2644) 
          javax.swing.JTabbedPane.fireStateChanged(JTabbedPane.java:394) 
          javax.swing.JTabbedPane$ModelListener.stateChanged(JTabbedPane.java:270) 
          javax.swing.DefaultSingleSelectionModel.fireStateChanged(DefaultSingleSelectionModel.java:132) 
          javax.swing.DefaultSingleSelectionModel.setSelectedIndex(DefaultSingleSelectionModel.java:67) 
          javax.swing.JTabbedPane.setSelectedIndexImpl(JTabbedPane.java:616) 
          javax.swing.JTabbedPane.setSelectedIndex(JTabbedPane.java:591) 
          javax.swing.plaf.basic.BasicTabbedPaneUI$Handler.mousePressed(BasicTabbedPaneUI.java:3647) 
          java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:280) 
          java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279) 
          java.awt.Component.processMouseEvent(Component.java:6522) 
          javax.swing.JComponent.processMouseEvent(JComponent.java:3324) 
          java.awt.Component.processEvent(Component.java:6290) 
          java.awt.Container.processEvent(Container.java:2234) 
          java.awt.Component.dispatchEventImpl(Component.java:4881) 
          java.awt.Container.dispatchEventImpl(Container.java:2292) 
          java.awt.Component.dispatchEvent(Component.java:4703) 
          java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898) 
          java.awt.LightweightDispatcher.processMouseEvent(Container.java:4530) 
          java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462) 
          java.awt.Container.dispatchEventImpl(Container.java:2278) 
          java.awt.Window.dispatchEventImpl(Window.java:2750) 
          java.awt.Component.dispatchEvent(Component.java:4703) 
          java.awt.EventQueue.dispatchEventImpl(EventQueue.java:751) 
          java.awt.EventQueue.access$500(EventQueue.java:97) 
          java.awt.EventQueue$3.run(EventQueue.java:702) 
          java.awt.EventQueue$3.run(EventQueue.java:696) 
          java.security.AccessController.doPrivileged(Native Method) 
          java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75) 
          java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86) 
          java.awt.EventQueue$4.run(EventQueue.java:724) 
          java.awt.EventQueue$4.run(EventQueue.java:722) 
          java.security.AccessController.doPrivileged(Native Method) 
          java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75) 
          java.awt.EventQueue.dispatchEvent(EventQueue.java:721) 
          java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201) 
          java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116) 
          java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105) 
          java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) 
          java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93) 
          java.awt.EventDispatchThread.run(EventDispatchThread.java:82) 
    (Listeners should be removed in the plugin dispose method!) 

```

**Comment by lukasged** [comment]
You tried with the `v1.0.0` release, right? I had this issue resolved in this [commit](https://github.com/lukasged/oops-plugin/commit/d0c9ec4de6a31e584abd9c50e0c58f6101935651) and now released the `v1.0.1` with the issue resolved ✔️

**Comment by csnyulas** [comment]
Yes, the problem was with 1.0.0.
1.0.1 works fine, indeed, and since you modified the 'update.properties' file Protégé picks up correctly the latest version.
